src/main/docs/guide/httpServer/errorHandling/localErrorHandling.adoc

```diff
@@ -11,3 +11,8 @@ snippet::io.micronaut.docs.server.json.PersonController[tags="statusError", inde
 <1> The api:http.annotation.Error[] declares which api:http.HttpStatus[] error code to handle (in this case 404)
 <2> A api:http.hateoas.JsonError[] instance is returned for all 404 responses
 <3> An api:http.HttpStatus#NOT_FOUND[] response is returned
+
+Similar to other controller methods, error handlers can use <<binding, request binding annotations>> on parameters e.g.
+to access header values. However, binding the request body comes with additional restrictions depending on the HTTP
+server implementation used. If the body has already been bound to a parameter of the original controller method, it may
+not be possible to bind the body to a different type, as the original bytes may already have been discarded.
```

Comment:
Can you add that for conversion error you can bind the body as a method parameter with `@Body String body`?

Reply:
i dont think we can guarantee this for e.g. servlet. and it may not be the case anymore if we move the netty parsing code to loom at some point

Reply:
or to be more specific: this works right now in netty, because we basically buffer the full input bytes, and then transform them to the object as one, so we still have the bytes if the transform fails. in servlet, you transform an input stream, and dont keep the bytes around. this may also be a desirable approach for netty if it can be achieved without context switching (which will need some loom work).

Reply:
If I am understanding correctly. In Micronaut Framework 4, we no longer support accessing the body in an exception handler. At least, not in a reliable way. is that correct?

Reply:
it was not reliable before either, i dont think accessing the body as string would work if there was no conversion error in 3.x either.